layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,42 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode 
+                      f 
+                      dash-functional
```

Comment:
I tried to remove these lines and the mode wouldn't start since it couldn't find `dash-functional` and `f`. Keep in mind that `dash` is a different package to `dash-functional`, and so installing one package doesn't install the other. Further, since the symbol `dash-functional` doesn't appear anywhere in Spacemacs's codebase, it isn't installed. For similar reasons, I don't think that `f` is installed by default.